.ci/pytorch/build.sh

```diff
@@ -263,7 +263,7 @@ if [[ "$BUILD_ENVIRONMENT" == *-bazel-* ]]; then
     # Build torch, the Python module, and tests for CPU-only
     tools/bazel build --config=no-tty "${BAZEL_MEM_LIMIT}" "${BAZEL_CPU_LIMIT}" --config=cpu-only :torch :torch/_C.so :all_tests
   else
-    tools/bazel build --config=no-tty "${BAZEL_MEM_LIMIT}" "${BAZEL_CPU_LIMIT}" //...
```

Comment:
This recursively builds third_party directory with bazel. For some reason I don't fully understand, bazel tries to build go and typescript packages `third_party/flatbuffers` (and failed on this PR but not master??). The change here limit the scope of the build to all main targets.